examples/bigcommerce/lib/bigcommerce/bigcommerce.ts

```diff
@@ -0,0 +1,195 @@
+const STORE_NAME = process.env.BIGCOMMERCE_STORE_NAME
+const ACCESS_TOKEN = process.env.BIGCOMMERCE_ACCESS_TOKEN
+
+const getBearerToken = async () => {
```

Comment:
Since we're using function declarations in other places I suggest we make this a function declaration as well. What do you think?